### PR TITLE
Added option to not show the same selected item in list of items and created label to dropdown

### PR DESCRIPTION
**Pull request by lucas-sesti**
I created a logic to not show the same selected item, example:
Low
Medium
High

when Low is selected, the item list will be: Medium and High, and if i selected Medium, the list will be: High, Low, etc...

and i created a label with some options, like background color, vertical position, and left position..

### CODE REVIEW

File: lib/menu_button.dart [hunk 1]
```diff
@@ -59,7 +195,7 @@ class _MenuButtonState<T> extends State<MenuButton<T>> {
     final RenderBox overlay = Overlay.of(context).context.findRenderObject();
     final RelativeRect position = RelativeRect.fromRect(
       Rect.fromPoints(
-        button.localToGlobal(const Offset(0, 0), ancestor: overlay),
+        button.localToGlobal(const Offset(0, 12), ancestor: overlay),
```
Comment (huextrat) [inline lib/menu_button.dart:198]: I'm not sure about the new Offset it should be `(0, 0)` to keep the Rect aligned
```suggestion
        button.localToGlobal(const Offset(0, 0), ancestor: overlay),
```

Comment (lucas-sesti) [reply lib/menu_button.dart:198]: I forgot change the 12, my mistake, sorry, but the 12 is the verticalContentPadding from LabelDecoration,  (if have a label decoration), if don't have a label decoration will be 0

**Comment by lucas-sesti** [comment]
You will update the offset or you want i change here and commit again?

**Comment by huextrat** [comment]
> You will update the offset or you want i change here and commit again?

It's should be done :)

**Comment by huextrat** [comment]
Thanks a lot for all contributions @lucas-sesti :)

**Pull request merged**